sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -555,12 +555,44 @@ public void delete(String contactUuid, DeletionDetails deletionDetails) {
 		deleteContact(contact, deletionDetails);
 	}
 
+	@Override
+	@RightsAllowed(UserRight._CONTACT_DELETE)
+	public void delete(List<String> uuids, DeletionDetails deletionDetails) {
+		deleteContacts(uuids, deletionDetails);
+	}
+
 	@Override
 	@RightsAllowed(UserRight._CONTACT_DELETE)
 	public void restore(String uuid) {
 		super.restore(uuid);
 	}
 
+	@Override
+	@RightsAllowed(UserRight._CONTACT_DELETE)
+	public void restore(List<String> uuids) {
+		restoreContacts(uuids);
+	}
+
+	@Override
+	@RightsAllowed(UserRight._CONTACT_DELETE)
+	public List<String> restoreContacts(List<String> uuids) {
```

Comment:
Since this implementation seems to be the same for all classes, please move this into a default implementation of the restore method in the `DeletableFacade`, and make the logged error more generic (i.e. replace the concrete entity name with "entity"; the entity in question will be pretty clear from the stack trace anyway)